src/types.ts

```diff
@@ -35,6 +35,13 @@ export type WebpubManifest = {
   spine: Spine<'text/html'>;
   links: any[];
   resources: any[];
+  readingOrder?: ReadingOrder[];
+};
+
+export type ReadingOrder = {
+  href: string;
+  title: string;
+  type: string;
```

Comment:
This is good, our WebpubManifest type is out of date actually. I generated a full type from the official specs in epub-to-webpub that I'm going to have Jiayong extract and we can then import it here and it will have full defs for `readingOrder` etc.